rootfs/etc/cont-init.d/08-svc-snmptrapd.sh

```diff
@@ -16,6 +25,15 @@ mkdir -p /run/snmptrapd
 chown -R librenms. /run/snmptrapd
 
 sed -ie "s/@LIBRENMS_SNMP_COMMUNITY@/${LIBRENMS_SNMP_COMMUNITY}/" /etc/snmp/snmptrapd.conf
+sed -ie "s/@SNMP_ACTION@/${SNMP_ACTION}/" /etc/snmp/snmptrapd.conf'
```

Comment:
To align with net-SNMP, `ACTION` should be renamed `PROCESSING_TYPE`

```suggestion
sed -ie "s/@SNMP_PROCESSING_TYPE@/${SNMP_PROCESSING_TYPE}/" /etc/snmp/snmptrapd.conf'
```